client/gatsby-browser.js

```diff
@@ -30,9 +30,6 @@ export const wrapPageElement = layoutSelector;
 export const disableCorePrefetching = () => true;
 
 export const onClientEntry = () => {
-  // purge the csrf cookies, rather than relying what the browser decides a
-  // Session duration is
-  cookies.erase('_csrf');
```

Comment:
> Would you mind explaining this?

Not at all. Since this PR sets `_csrf` to be `httpOnly` it's no-longer possible for JS to manipulate the cookie at all. That includes resetting it like we were doing here.